webapp/authentication.py

```diff
@@ -39,6 +39,7 @@ def empty_session(session):
     session.pop("macaroon_discharge", None)
     session.pop("openid", None)
     session.pop("user_shared_snaps", None)
+    session.pop("github_auth_secret", None)
```

Comment:
should this be in here? Is @jkfran doing it in another branch? If it's already done rebase - as adding this line seems to lower code coverage